drivers/media/i2c/ov5647.c

```diff
@@ -152,7 +156,7 @@ static struct regval_list ov5647_640x480_8bit[] = {
 	{0x3036, 0x46},
 	{0x303c, 0x11},
 	{0x3106, 0xf5},
-	{0x3821, 0x07},
+	{0x3821, 0x03},
```

Comment:
I think my only comment is that we could drop bit 1 from these registers totally, as the control handler will always be called via __v4l2_ctrl_handler_setup and set it as defined by the control. That only leaves the binning bit defined.

Reply:
My only thought was for the register set on its own to correspond to "no flips at all", which, bizarrely, involves setting the hflip bit to counteract the secret internal flip. But given that we don't document anything about the registers anyway - I don't suppose it makes any difference really!